cms/djangoapps/contentstore/views/course.py

```diff
@@ -1126,11 +1146,12 @@ def settings_handler(request, course_key_string):
         elif 'application/json' in request.META.get('HTTP_ACCEPT', ''):
             if request.method == 'GET':
                 course_details = CourseDetails.fetch(course_key)
-                return JsonResponse(
-                    course_details,
-                    # encoder serializes dates, old locations, and instances
-                    encoder=CourseSettingsEncoder
```

Comment:
is this removed intentionally?

Reply:
CourseSetting encoder is doing the same thing.

Reply:
I want to send some extra data in Json response.